pkg/shell/nav.jsx

```diff
@@ -48,6 +48,7 @@ export class CockpitNav extends React.Component {
         // Click on active menu item (when using arrows to navigate through menu)
         function clickActiveItem() {
             const cur = document.activeElement;
+            console.log("This wont run");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17475-20220627-130821-94ed2deb-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)